cmd/notation/sign.go

```diff
@@ -189,5 +215,23 @@ func prepareSigningOpts(opts *signOpts) (notation.SignOptions, error) {
 		},
 		UserMetadata: userMetadata,
 	}
+	if opts.tsaServerURL != "" {
+		// timestamping
+		fmt.Printf("Timestamping with TSA %q\n", opts.tsaServerURL)
+		signOpts.Timestamper, err = tspclient.NewHTTPTimestamper(&http.Client{Timeout: TimestampingTimeout}, opts.tsaServerURL)
```

Comment:
Looks like we cannot use the `auth.Client` here, because the `tspclient.NewHTTPTimestamper` takes an `*http.Client` as input.

Since the OCSP client has the same issue, I will create an issue to track it. And we could fix it in the next iteration.

Reply:
Issue created: https://github.com/notaryproject/notation/issues/988

Reply:
Based on offline discussion, updated the `http.Client` in this PR.